content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -0,0 +1,297 @@
+
+{{< tabs >}}
+
+{{% tab "Environment variables" %}}
+
+To configure the tracer using environment variables, set the variables before launching the instrumented application. To learn how to set environment variables in different environments, see [Configuring process environment variables][1].
+
+[1]: /tracing/trace_collection/dd_libraries/dotnet-core/#configuring-process-environment-variables
+{{% /tab %}}
+
+{{% tab "Code" %}}
+
+To configure the Tracer in application code, create a `TracerSettings` instance from the default configuration sources. Set properties on this `TracerSettings` instance before calling `Tracer.Configure()`. For example:
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> Settings must be set on <code>TracerSettings</code> <em>before</em> creating the <code>Tracer</code>. Changes made to <code>TracerSettings</code> properties after the <code>Tracer</code> is created are ignored.
+</div>
+
+```csharp
+using Datadog.Trace;
+using Datadog.Trace.Configuration;
+
+// read default configuration sources (env vars, web.config, datadog.json)
+var settings = TracerSettings.FromDefaultSources();
+
+// change some settings
+settings.Environment = "prod";
+settings.ServiceName = "MyService";
+settings.ServiceVersion = "abc123";
+settings.Exporter.AgentUri = new Uri("http://localhost:8126/");
+
+// configure the global Tracer settings
+Tracer.Configure(settings);
+```
+
+{{% /tab %}}
+
+{{% tab "JSON file" %}}
+
+To configure the Tracer using a JSON file, create `datadog.json` in the instrumented application's directory. The root JSON object must be an object with a key-value pair for each setting. For example:
+
+```json
+{
+  "DD_TRACE_AGENT_URL": "http://localhost:8126",
+  "DD_ENV": "prod",
+  "DD_SERVICE": "MyService",
+  "DD_VERSION": "abc123",
+}
+```
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+### Configuration settings
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> On Linux, the names of environment variables are case-sensitive.
+</div>
+
+Using the methods described above, customize your tracing configuration with the following variables. Use the environment variable name (for example, `DD_TRACE_AGENT_URL`) when setting environment variables or configuration files. Use the TracerSettings property (for example, `Exporter.AgentUri`) when changing settings in code.
+
+#### Unified Service Tagging
+
+To use [Unified Service Tagging][4], configure the following settings for your services:
+
+`DD_ENV`
+: **TracerSettings property**: `Environment`<br>
+If specified, adds the `env` tag with the specified value to all generated spans. Added in version 1.17.0.
+
+`DD_SERVICE`
+: **TracerSettings property**: `ServiceName`<br>
+If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from application name (IIS application name, process entry assembly, or process name). Added in version 1.17.0.
```

Comment:
```suggestion
If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from the application name (IIS application name, process entry assembly, or process name). Added in version 1.17.0.
```